src/sagemaker/lineage/query.py

```diff
@@ -226,29 +226,102 @@ def __init__(
 
     def __str__(self):
         """Define string representation of ``LineageQueryResult``.
-        
+
         Format:
         {
             'edges':[
                 {
-                    'source_arn': 'string', 'destination_arn': 'string', 
+                    'source_arn': 'string', 'destination_arn': 'string',
                     'association_type': 'string'
                 },
                 ...
             ]
             'vertices':[
                 {
-                    'arn': 'string', 'lineage_entity': 'string', 
-                    'lineage_source': 'string', 
+                    'arn': 'string', 'lineage_entity': 'string',
+                    'lineage_source': 'string',
                     '_session': <sagemaker.session.Session object>
                 },
                 ...
             ]
         }
-        
+
         """
         result_dict = vars(self)
-        return (str({k: [vars(val) for val in v] for k, v in result_dict.items()}))
+        return str({k: [vars(val) for val in v] for k, v in result_dict.items()})
+
+    def _import_visual_modules(self):
+        """Import modules needed for visualization."""
+        import dash_cytoscape as cyto
+
+        from jupyter_dash import JupyterDash
+
+        from dash import html
+
+        return cyto, JupyterDash, html
+
+    def _get_verts(self):
+        """Convert vertices to tuple format for visualizer"""
+        verts = []
+        for vert in self.vertices:
+            verts.append((vert.arn, vert.lineage_source))
+        return verts
+
+    def _get_edges(self):
+        """Convert edges to tuple format for visualizer"""
+        edges = []
+        for edge in self.edges:
+            edges.append((edge.source_arn, edge.destination_arn))
```

Comment:
Should this include the `edge.association_type`?

Reply:
Included edge.association_type in the current commit, also made some graph style changes.